conf/machine/openvario-57-lvds_DS2.conf

```diff
@@ -43,7 +43,7 @@ MACHINE_FEATURES ?= "alsa apm keyboard rtc serial screen usbgadget usbhost vfat"
 
 SOC_FAMILY = "sun7i"
 
-KERNEL_DEVICETREE_SOURCE = "openvario-7-AM070.dts"
+KERNEL_DEVICETREE_SOURCE = "openvario-57-lvds_DS2.dts"
```

Comment:
The name is separated by dashes, let's keep it consistent: `openvario-57-lvds-DS2`. Don't mix underscores and dashes: makes it hard to type correctly and causes confusion.